content/blog/posts/o3de-interactive-tutorials-prototyping-2022.md

```diff
@@ -0,0 +1,40 @@
+---
+title: "Advance Prototyping of the InteractiveTutorials Gem for O3DE"
+description: ""
+date: 2022-08-10
+slug: ""
+author: Madeleine Song
+blog_img: "/images/blog/interactive-tutorials-prototyping/interactive-tutorials.png"
+full_img: "/images/blog/interactive-tutorials-prototyping/interactive-tutorials.png"
+---
+Hi! I’m Madeleine, an Amazon intern working on Open 3D Engine (O3DE). My summer project focused on advanced prototyping of the InteractiveTutorials gem. So far, I've added several features and scoped out prospective future improvements. I worked on automating and making tutorials easier to navigate.
```

Comment:
```suggestion
Hi! I’m Madeleine, an Amazon intern working on Open 3D Engine (O3DE). My summer project focused on advanced prototyping of the InteractiveTutorials gem. So far, I've added several features and scoped out prospective future improvements. I also worked on automating and making tutorials easier to navigate.
```

I think adding `also` here makes the distinction that this is in addition to the features you added and scoping out of future improvements.